WatchApp Extension/ComplicationController.swift

```diff
@@ -81,14 +81,19 @@ final class ComplicationController: NSObject, CLKComplicationDataSource {
         let template: CLKComplicationTemplate?
         
         let glucoseText = CLKSimpleTextProvider.localizableTextProvider(withStringsFileTextKey: "120↘︎", shortTextKey: "120")
-        let timeText = CLKTimeTextProvider(date: Date())
+        let timeText = CLKRelativeDateTextProvider(date: Date(), style: .natural, units: .minute)
```

Comment:
I believe it does. It is reactive to available space so it should fit. I haven't ever been able to get the utilitarian large complication to work to test that one, nor have I looked into why. This change definitely works for modular small as well as the large version though.